titiler/pgstac/factory.py

```diff
@@ -554,10 +553,27 @@ def wmts(
             ]
             if qs:
                 tiles_url += f"?{urlencode(qs)}"
+            layers.append({"name": "default", "endpoint": tiles_url})
```

Comment:
We always add a `default` layer, but this one will likely not work if no query-parameters are passed

That's said, it might be possible that some user use custom readers that do not need `assets` input 🤷‍♂️